packages/react-tinacms-github/src/github-client/authenticate.ts

```diff
@@ -34,17 +34,21 @@ export const authenticate = (
     let authTab: Window | undefined
     window.addEventListener('storage', function(e: StorageEvent) {
       if (e.key == GITHUB_AUTH_CODE_KEY) {
-        fetch(
-          `${codeExchangeRoute}?code=${e.newValue}&state=${authState}`
-        ).then(() => {
-          if (authTab) {
-            authTab.close()
-          }
-          resolve()
-        })
+        fetch(`${codeExchangeRoute}?code=${e.newValue}&state=${authState}`)
+          .then(response => response.json())
+          .then(data => {
+            const token = data.signedToken || null
+            if (token) {
```

Comment:
Do you think we should plan to remove this check in the future? If so, we could add a deprecation warning in the console